app/src/main/res/layout/strip_container.xml

```diff
@@ -24,16 +24,23 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         style="?attr/suggestionStripViewStyle" />
-    <LinearLayout
-        android:id="@+id/clipboard_strip"
-        android:orientation="horizontal"
+    <HorizontalScrollView
+        android:id="@+id/clipboard_strip_scroll_view"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        style="?attr/suggestionStripViewStyle" >
-        <!-- empty view to move buttons to the right -->
-        <View
-            android:layout_width="0dp"
-            android:layout_height="0dp"
-            android:layout_weight="1" />
-    </LinearLayout>
+        android:visibility="gone"
```

Comment:
Why did you add `android:visibility="gone"`? Is it necessary now for some reason?

Reply:
no I don't think it is necessary. I added it while debugging and probably forgot to remove it